TrafficCapture/dockerSolution/src/main/docker/docker-compose.yml

```diff
@@ -95,6 +95,8 @@ services:
       - migrations
     volumes:
       - sharedReplayerOutput:/shared-replayer-output
+      - ./migrationConsole/console_link/services.yaml:/etc/migration_services.yaml
+      - ./migrationConsole/console_link:/root/console_link
```

Comment:
Is this needed since image already has the library?

Reply:
No, it's not needed, but it means that I can make code changes locally and instantly test it on the docker container. What do you think about making a note that this is a temporary thing, but keeping it in for now? This seems like the sort of thing that's okay on the e2e branch, but I want a reminder to take it out before we merge to main

Reply:
Ah gotcha, yeah I'm fine with a comment for now